tests/models/seamless_m4t/test_modeling_seamless_m4t.py

```diff
@@ -414,9 +414,11 @@ def _get_encoder_outputs(
         encoder_outputs["last_hidden_state"] = encoder_outputs.last_hidden_state.repeat_interleave(
             num_interleave, dim=0
         )
+        generation_config = copy.deepcopy(model.generation_config)
+        generation_config = model._prepare_special_tokens(generation_config)
```

Comment:
IMO feels a bit weird that preparing the special tokens returns a genertaion config. Would maybe not return and do inplace